src/java/picard/util/IntervalListScatterer.java

```diff
@@ -60,10 +60,14 @@ public int compare(final Interval o1, final Interval o2) {
         }
     }
 
-    public List<IntervalList> scatter(final IntervalList sourceIntervalList, final int scatterCount) {
+    public List<IntervalList> scatter(final IntervalList uniquedIntervalList, final int scatterCount) {
+        return scatter(uniquedIntervalList, false, scatterCount);
+    }
+
+    public List<IntervalList> scatter(final IntervalList sourceIntervalList, final boolean isUniqued, final int scatterCount) {
```

Comment:
Could the ```isUniqued``` parameter be the third argument so that it looks like you appended to line 63?